### PR TITLE
Api work get 

### PR DESCRIPTION
**Pull request by katerina-aa**

### CODE REVIEW

File: spec/api/api_client.rb [hunk 1]
```diff
@@ -11,6 +12,16 @@ def get_user(username)
...
 
+  def generate_random_long_username
+    username = SecureRandom.alphanumeric(256)
+  end
+
+  def generate_random_symbol_username
+    symbols = ['!', '@', '$', '%', '#', '^', '&', '*', '(', ')', '_', '=', '+']
+    username = ''
+    rand(4..16).times { username << symbols[rand(0...symbols.size)] }
+  end
```
Comment (AndriiMish) [inline spec/api/api_client.rb]: these methods are better to be placed in spec, not in client class

File: spec/api/api_get_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+require_relative 'api_client.rb'
+require 'securerandom'
+
+RSpec.describe 'GET /user/username' do
+
+  app_cl = ApiClient.new
+  body = app_cl.generate_random_body
+  
+  before(:all) { app_cl.create_user(body) }
+  after(:all) { app_cl.delete_user(body[:username]) }
```
Comment (AndriiMish) [inline spec/api/api_get_spec.rb:29]: nice!

File: spec/api/api_get_spec.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  end
+
+ hh_usernames = [ 
+   { "username" => app_cl.generate_random_long_username }, 
+   { "username" => app_cl.generate_random_symbol_username } 
+  ]
+
+  hh_usernames.each do | hh |
+
```
Comment (AndriiMish) [inline spec/api/api_get_spec.rb]: remove empty line pls

File: spec/api/api_get_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      response = app_cl.get_user(body[:username])
+      expect(response.status).to eq(200)
+    end
+
+  end
+
+ hh_usernames = [ 
+   { "username" => app_cl.generate_random_long_username }, 
+   { "username" => app_cl.generate_random_symbol_username } 
+  ]
```
Comment (AndriiMish) [inline spec/api/api_get_spec.rb]: goooood!!

**Comment by AndriiMish** [comment]
resolve conflict please

**Comment by katerina-aa** [comment]
I created a new pull request without conflicts :)

**Pull request closed without merging**